docker/deploy-fedimintd/deploy.sh

```diff
@@ -106,10 +106,13 @@ ssh -q "root@$ssh_host" << EOF
 
   sed -i 's/my-super-host.com/$domain/g' $host_dir/.env
 
+  ufw allow 8173/tcp
+  ufw allow 8174/tcp
```

Comment:
I thought it was udp:

https://github.com/fedimint/fedimint/blob/c5293810359a06887eccb817cf8bafae6659c02b/docker/iroh-fedimintd/docker-compose.yaml#L6

Reply:
Right...